### ISSUE TITLE
Error running docker

### ISSUE THREAD

**Issue by tonila** [kfatehi/docker-sinopia#15]
Pull  

```
$ docker pull keyvanfatehi/sinopia:latest
```

Run  

```
$ docker run --name sinopia -d -p 4873:4873 keyvanfatehi/sinopia:latest
```

Check logs  

```
$ docker logs sinopia

module.js:340
    throw err;
          ^
Error: Cannot find module 'sinopia/lib/config_gen'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/opt/sinopia/config_gen.js:2:14)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
sed: can't read /tmp/config.yaml: No such file or directory
 warn  --- config file  - /opt/sinopia/config.yaml

/opt/sinopia/node_modules/sinopia/lib/cli.js:75
  if (!config.user_agent) config.user_agent = 'Sinopia/'+pkg.version
             ^
TypeError: Cannot read property 'user_agent' of undefined
    at afterConfigLoad (/opt/sinopia/node_modules/sinopia/lib/cli.js:75:14)
    at Object.<anonymous> (/opt/sinopia/node_modules/sinopia/lib/cli.js:58:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/opt/sinopia/node_modules/sinopia/bin/sinopia:3:1)
    at Module._compile (module.js:456:26)
```

**Comment by cmuller** [comment]
Hello,

I had the same error at the first docker run but then if I stop the container, edit a valid config file as explained in the README (e.g., based on https://github.com/rlidwka/sinopia/blob/master/conf/full.yaml), and start again, it seems to work ok.

Hope it'll help.

**Comment by osherx** [comment]
yup, we got the same error.

I think the root of it is that the way sinopia was configured and run has changed.

the whole `sinopia/lib/config_gen` is no longer necessary.

im new to docker, I'm not sure I understand everything down there, and there could be things that are no longer necessary but - 
playing with it mannually - I think I got it running with the following Dockerfile:
```
FROM nodejs/latest
MAINTAINER Keyvan Fatehi <keyvanfatehi@gmail.com>
RUN adduser --disabled-password --gecos "" sinopia
RUN mkdir -p                 /opt/sinopia/storage
RUN chown -R sinopia:sinopia /opt/sinopia
WORKDIR /opt/sinopia
RUN npm install -g sinopia
USER sinopia
CMD ["sinopia"]
EXPOSE 4873
VOLUME /opt/sinopia
```

`start.sh` is then no longer necessary

disclaimer - we don't use `FROM nodejs/latest`, we have a base of our own, but I don't think it's relevant, it's about our monitoring tools and stuff.
